packages/fiori/src/themes/Bar.css

```diff
@@ -2,8 +2,8 @@
 	background-color: var(--sapPageHeader_Background);
 	height: var(--_ui5_bar_base_height);
 	width: 100%;
-	box-shadow: var(--sapContent_HeaderShadow);
 	display: block;
+	box-shadow: var(--sapContent_HeaderShadow);
```

Comment:
put this line back to where it was, it doesn't make sense anymore to change anything in this file